docs/development.rst

```diff
@@ -1,6 +1,91 @@
 .. _development:
 
+1. System requirement :
 
+Any system with basic configuration.
+Operating System : Any (Windows / Linux / Mac).
+
+2. Software requirement :
+Updated browser
+python
+
+3. Skill set :
+Knowledge of git & github.
+python
+
+============================
+Setting Up local Environement
+============================
+
+============================
+Forking the Repo
+============================
+1.Firstly you have to make your own copy of project. For that you have to fork the repository. You can find the fork button on the top-right side of the browser window. 
+
+2.Kindly wait till it gets forked.
+
+3.After that copy will look like <your-user-name>/MSS forked from Open-MSS/MSS.
+
+============================
+Cloning the Repo
+============================
+
+1.Now you have your own copy of project. Here you have to start your work.
+
+2.Go to desired location on your computer where you want to set-up the project.
+
+3.Right click there and click on git bash. A terminal window will pop up
+
+4.Type the command git clone <your-fork-url>.git and hit enter.
```

Comment:
Git gives an easy reference to your repo and instructions how to clone. Please advise people to use that (The big green button with CODE).